flytekit/core/base_task.py

```diff
@@ -734,7 +737,8 @@ def dispatch_execute(
             #   a workflow or a subworkflow etc
             logger.info(f"Invoking {self.name} with inputs: {native_inputs}")
             with timeit("Execute user level code"):
-                native_outputs = self.execute(**native_inputs)
+                task = decorate_python_task(self)
+                native_outputs = task.execute(**native_inputs)
```

Comment:
Is the vscode decorator supposed to work with eager workflows?

Reply:
Not sure, I don't think we tested it before.